### PR TITLE
fix: tweak check-requirements for calling ldconfig

### LINKED ISSUE #9405: Connection to the host gets stuck at the stage of Downloading VS Code Server (Could not fetch remote environment).
Type: <b>Bug</b>

After recent VS Code Insiders 1.86.0 update, the Remote-SSH extension has stopped performing its functions of connecting to the host. 
During the connection, the process gets stuck at the Downloading VS Code Server stage, which is called several times. At the end of the process it gives an error:

Failed to connect to the remote extension host server (Error: WrappedError(WrappedError { message: "error checking server integrity", original: "failed to run command \"/home/user/.vscode-server-insiders/cli/servers/Insiders-0b571a26e213bf1ec6266e37152ced80c006bd1e.staging/server/bin/code-server-insiders --version\" (code 1): /home/user/.vscode-server-insiders/cli/servers/Insiders-0b571a26e213bf1ec6266e37152ced80c006bd1e.staging/server/bin/helpers/check-requirements.sh: 35: ldconfig: not found\n" })))

However, when I downgrade VS Code Insiders to 1.85.0, the extension works again without problems. 

Extension version: 0.107.1
VS Code version: Code - Insiders 1.86.0-insider (0b571a26e213bf1ec6266e37152ced80c006bd1e, 2024-01-18T05:36:10.559Z)
OS version: Windows_NT x64 10.0.22631
Modes:
Connection to 'SSH: arkham' could not be established  WrappedError(WrappedError { message: "error checking server integrity", original: "failed to run command \"/home/artem/.vscode-server-insiders/cli/servers/Insiders-0b571a26e213bf1ec6266e37152ced80c006bd1e.staging/server/bin/code-server-insiders --version\" (code 1): /home/artem/.vscode-server-insiders/cli/servers/Insiders-0b571a26e213bf1ec6266e37152ced80c006bd1e.staging/server/bin/helpers/check-requirements.sh: 35: ldconfig: not found\n" })

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 5 3500U with Radeon Vega Mobile Gfx   (8 x 2096)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|6.94GB (0.53GB free)|
|Process Argv|--crash-reporter-id 715bd2ab-bfab-4086-9413-dcb5f66d120d|
|Screen Reader|no|
|VM|0%|

Connection to 'SSH: arkham' could not be established  WrappedError(WrappedError { message: "error checking server integrity", original: "failed to run command \"/home/artem/.vscode-server-insiders/cli/servers/Insiders-0b571a26e213bf1ec6266e37152ced80c006bd1e.staging/server/bin/code-server-insiders --version\" (code 1): /home/artem/.vscode-server-insiders/cli/servers/Insiders-0b571a26e213bf1ec6266e37152ced80c006bd1e.staging/server/bin/helpers/check-requirements.sh: 35: ldconfig: not found\n" })
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805cf:30301675
vsaa593cf:30376535
py29gd2263:30784851
vsclangdf:30492506
c4g48928:30535728
vscrpc:30624061
pythongtdpath:30726887
welcomedialog:30812478
pythonidxpt:30768918
pythonnoceb:30776497
asynctok:30898717
dsvsc013:30777762
dsvsc014:30777825
dsvsc015:30821418
pythontestfixt:30866404
pythonregdiag2:30926734
pyreplss1:30879911
pythonmypyd1:30859725
pythoncet0:30859736
pythontbext0:30879054
accentitlementst:30870582
dsvsc016:30879898
dsvsc017:30880771
dsvsc018:30880772

```

</details>

### PR DESCRIPTION
**Pull request by Leask**
Sometimes, the target system has `ldconfig` ready, but you can not assume that `/sbin` is in $PATH. This may cause the installation of 'ssh remote' to fail.

Fixes https://github.com/microsoft/vscode-remote-release/issues/9405

### CODE REVIEW

**Comment by Leask** [comment]
Screenshot 2024-01-17 at 4 00 56 AM

Issue example on latest Debian release.

**Review by deepak1556 — APPROVED** [review]
Thanks!

File: resources/server/bin/helpers/check-requirements-linux.sh [hunk 1]
```diff
@@ -66,7 +66,7 @@ if [ -n "$(ldd --version | grep -v musl)" ]; then
         libc_path='/usr/lib/libc.so.6'
     elif [ -f /sbin/ldconfig ]; then
         # Look up path
-        libc_paths=$(ldconfig -p | grep 'libc.so.6')
+        libc_paths=$(/sbin/ldconfig -p | grep 'libc.so.6')
```
Comment (deepak1556) [inline resources/server/bin/helpers/check-requirements-linux.sh:69]: Can you also make the change at L35 https://github.com/microsoft/vscode/blob/969393c1a6d24c25e72cb7ba4f918cb13b9263a2/resources/server/bin/helpers/check-requirements-linux.sh#L35

**Review by deepak1556 — CHANGES_REQUESTED** [review]

**Comment by Leask** [comment]
@deepak1556 🤝 Done! Thanks for reminding me!

**Review by deepak1556 — APPROVED** [review]

**Review by roblourens — APPROVED** [review]

**Comment by Leask** [comment]
Cool! 😄

**Pull request merged**